client-apis/decrypt.md

```diff
@@ -7,6 +7,16 @@
 
 0.1.0-preview
 
+### Changelog
+
+- 0.2.0
+
+  - [Detect Base64-encoded Messages](../changes/2020-05-13_remove-keyring-trace/change.md)
```

Comment:
Should this be `../changes/2020-07-13_detect-base64-encoded-messages/change.md` ?

Reply:
Yup, copy/paste error.